docs/symphony-book/ai-management/parameter-management.md

```diff
@@ -55,7 +55,7 @@ spec:
     cv-skill.def.recording_duration: "34"
 ```
 
-When an application queries AI skill through [Symphony agent](../agent/agent.md), the following rules apply:
+When an application queries AI skill through [Symphony agent](../agent/_overview.md), the following rules apply:
```

Comment:
Curious - what's the benefit of using an underscore in name "_overview.md"

Reply:
Since we don't have a formal TOC structure where we can put our articles in the order we want them to appear, the underscore makes sure that the overview topic always shows up at the top of any set of articles in a folder.